__tests__/app.test.js

```diff
@@ -32,4 +32,28 @@ it('responds with 404 error if not path is found', ()=>{
     })
 });
   })
+  describe('/api/articles/:article_id', ()=>{
+    test('responds with an article object', ()=>{
+      return request(app).get('/api/articles/1').expect(200).then((response)=>{
+         expect(response.body.article).toHaveLength(1)
+        response.body.article.forEach((article) =>{
+          expect(article).toEqual(
+            expect.objectContaining({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
```

Comment:
you know exactly which article id should be coming back in this test, it would be good to be specific